Public/Sdk/Public/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -203,7 +203,8 @@ export function runQTest(args: QTestArguments): Result {
         Cmd.option("--qTestAttemptCount ", args.qTestAttemptCount),
         Cmd.option("--qTestTimeoutSec ", args.qTestTimeoutSec),
         Cmd.option(
-            "--vstestSettingsFile ", 
+            "--vstestSettingsFile ", qCodeCoverageEnumType === dynamicCodeCovString ? 
+            Artifact.input(args.vstestSettingsFileForCoverage) : 
             Artifact.input(args.vstestSettingsFile)

```

Comment:
I personally don't like this style of ternary code -- it's non standard.

Either
```
 qCodeCoverageEnumType === dynamicCodeCovString ?  Artifact.input(args.vstestSettingsFileForCoverage)  :  Artifact.input(args.vstestSettingsFile)
```
or
```
 qCodeCoverageEnumType === dynamicCodeCovString 
     ?  Artifact.input(args.vstestSettingsFileForCoverage)  
     :  Artifact.input(args.vstestSettingsFile)
```